config/diaspora.toml.example

```diff
@@ -285,10 +285,19 @@
 ## follow an account upon creation.
 #autofollow_on_join = true
 
-## Auto-follow account (default="hq@pod.diaspora.software")
+## Auto-follow accounts (default=["hq@pod.diaspora.software"])
+## A list of accounts to auto-follow when a user joins the pod. By default
+## the user follows the the diaspora* HQ account which keeps users up to
+## date with news about Diaspora. Feel free to add your podmin account to
+## the list.
+#autofollow_on_join_accounts= ["hq@pod.diaspora.software"]
+
+## Auto-follow account [To be deprecated] (default="hq@pod.diaspora.software")
 ## The diaspora* HQ account keeps users up to date with news about Diaspora.
 ## If you set another auto-follow account (for example your podmin account),
 ## please consider resharing diaspora* HQ's posts for your pod's users!
+## This config option will be deprecated in the next major release, use
+## `autofollow_on_join_accounts` instead.
 #autofollow_on_join_user = "hq@pod.diaspora.software"
```

Comment:
I feel like there's no real reason to deprecate this in the configuration example. Yes the code should still read it and print a deprecation warning if present (some check at boot time, not during runtime/actual use of the setting), but just dropping the setting from the example seems fine since the base functionality is the same with the new setting. A changelog entry describing the change is warranted of course.

Reply:
Resolved here: [16f699b](https://github.com/diaspora/diaspora/pull/8430/commits/16f699b52af1b4720b84fbbad8d27ef2492133e7).